jerry-core/ecma/operations/ecma-conversion.c

```diff
@@ -125,6 +125,18 @@ ecma_op_same_value (ecma_value_t x, /**< ecma value */
     return ecma_compare_ecma_strings (x_str_p, y_str_p);
   }
 
+#if ENABLED (JERRY_BUILTIN_BIGINT)
+  if (ecma_is_value_bigint (x))
+  {
+    if (!ecma_is_value_bigint (y))
+    {
+      return false;
+    }
+
+    return (ecma_bigint_compare_to_bigint (x, y) == 0);
```

Comment:
```c
return ecma_is_value_bigint (y) && ecma_bigint_compare_to_bigint (x, y) == 0;
```

Reply:
The 'if' above checks `ecma_is_value_bigint (y)`. Am I missing something?

Reply:
Or do you recommend to remove the `if`?

Reply:
Yep the request was only about to remove the if and combine it into the return condition.